cogs/event_handler.py

```diff
@@ -100,6 +106,33 @@ async def on_message(self, message):
                     delete_after=600,
                     view=hypixel_ping_buttons(),
                 )
+         @commands.Cog.listener()
+    async def on_message(self, message):
```

Comment:
Remove the duplicate on_message function

Reply:
ok tnx